lib/worker.coffee

```diff
@@ -20,7 +20,7 @@ module.exports = (text, attributes, options) ->
 
   Opal.ENV['$[]=']('PWD', path.dirname(options.opalPwd))
```

Comment:
I'm pretty sure we had only set this to workaround a bug in Opal. If Opal has resolved it, then I don't think we need it anymore.